tests/gdsconv/black_box_test/test_black_box.py

```diff
@@ -19,7 +19,7 @@
 if 'ALIGN_WORK_DIR' in os.environ:
     ALIGN_WORK_DIR = pathlib.Path(os.environ['ALIGN_WORK_DIR']).resolve()
 
-@pytest.mark.nightly
+@pytest.mark.skip
```

Comment:
Why skip this test? Please add a reason or bring it back.

Reply:
Sounds good. Could you please just annotate reason to skip then?

Reply:
Please check the latest commit that resolves all the reviews.